app/assets/javascripts/snowcrash/modules/table.js.coffee

```diff
@@ -0,0 +1,192 @@
+class IndexTable
```

Comment:
It would be nice to add some documentation/comments here.
This class depends on some ids/classes being present in the html code, like:
* `#index-table`
* `.js-select-all-items`
* `.js-table-actions`
* `js-multicheck`
* ...
(*IDEA: we could use a common namespace for all those dependencies, like starting all of them with `js-inde-table-...`*)

Reply:
✅